src/hyperlane/HyperlaneHelper.sol

```diff
@@ -33,6 +33,10 @@ contract HyperlaneHelper is Test {
         _help(mailbox, dispatchSelector, forkId, logs, enableEstimates);
     }
 
+    function findLogs(Vm.Log[] calldata logs, uint256 length) external pure returns (Vm.Log[] memory hlLogs) {
```

Comment:
It could be useful to have another function where you can specify the selector, like
`function findLogs(Vm.Log[] calldata logs, bytes32 dispatchSelector, uint256 length) ...`

Reply:
Sure, is there other selectors we should be aware about for LZ/HL?